build/configs/sidk_s5jt200/tc/defconfig

```diff
@@ -933,3 +933,4 @@ CONFIG_STACKMONITOR_PRIORITY=100
 CONFIG_STACKMONITOR_INTERVAL=5
 CONFIG_ENABLE_UPTIME=y
 # CONFIG_SYSTEM_VI is not set
+CONFIG_KERNEL_TEST_FW=y
```

Comment:
Don't modify this manually. Please update it through menuconfig.

Reply:
Ok done